CONTRIBUTING.md

```diff
@@ -26,3 +26,169 @@ information on using pull requests.
 
 This project follows [Google's Open Source Community
 Guidelines](https://opensource.google/conduct/).
+
```

Comment:
This doc needs Wojtek or Thinh to review

Reply:
@hejnal @thinhha :-)